kedro/framework/cli/pipeline.py

```diff
@@ -101,6 +101,11 @@ def pipeline():
     """Commands for working with pipelines."""
 
 
+@pipeline_cli.group()
+def micropkg():
+    """Commands for micropackaging."""
```

Comment:
We should separate this in its own file. We tend to have one `.py` file per CLI group (e.g. jupyter, pipeline. etc.), so this would keep it nice and tidy.

Reply:
It would also not be piggy-backing on "pipeline_cli" which makes it confusing to read.